code/_onclick/item_attack.dm

```diff
@@ -124,10 +124,12 @@
 				if(get_dist(get_turf(user), get_turf(M)) <= user.used_intent.reach)
 					user.do_attack_animation(M, visual_effect_icon = user.used_intent.animname)
 			return
-	if(istype(user.rmb_intent, /datum/rmb_intent/strong))
-		user.rogfat_add(10)
-	if(istype(user.rmb_intent, /datum/rmb_intent/swift))
-		user.rogfat_add(10)
+	var/rmb_stam_penalty = 1
```

Comment:
why is this 1? it should be 0 if you aren't using strong or swift

Reply:
didn't realize which file it was, fixed